pages/index.js

```diff
@@ -13,25 +15,36 @@ import ScrollBtn from '../app/components/scroll-btn';
 import store from '../app/data/index';
 import 'what-input';
 
-const Home = () => {
-  const baseUrl = `static/img/${store.currentYear}`;
-  return (
-    <Layout>
-      <main>
-        <Header />
-        <Updates />
-        <Speakers year="2019" baseUrl={baseUrl} speakers={store.data2019.speakers} />
-        {/* <Schedule /> */}
-        <Mission />
-        <Sponsors baseUrl={baseUrl} sponsors={store.data2019.sponsors} />
-        <Partners baseUrl={baseUrl} partners={store.data2019.partners} />
-        <Communities baseUrl={baseUrl} communities={store.data2019.communities} />
-        <Organizers />
-        <Venue />
-        <ScrollBtn />
-      </main>
-    </Layout>
-  );
-};
+class Home extends Component {
+  componentDidMount() {
```

Comment:
we should move this to Layout, as there are multiple pages that can load in next, but the Layout is always there

Reply:
ok, will make the change soon. thanks!

Reply:
Now done.